integration/fuse/src/main/java/alluxio/fuse/file/FuseFileInStream.java

```diff
@@ -84,24 +85,29 @@ private FuseFileInStream(FileInStream inStream, long fileLength, AlluxioURI uri)
 
   @Override
   public synchronized int read(ByteBuffer buf, long size, long offset) {
+    Preconditions.checkArgument(size >= 0 && offset >= 0 && size <= buf.capacity(),
```

Comment:
If a file is opened multiple times, the same iostream is shared, right?

Reply:
in fuse.open() we will assign an open file id
if multiple fuse.open() are issued, multiple io stream will be created
if only one open() is issued, the iostream will be used (not quite sure whether multiple threads will share the same open() or not)